src/app/win/Whoa.cpp

```diff
@@ -1,6 +1,6 @@
 #include "client/Client.hpp"
```

Comment:
```suggestion
#include "client/Client.hpp"
#include <windows.h>
```

Let's explicitly add `#include <windows.h>` here instead of relying on it coming in indirectly.